R/species_list.R

```diff
@@ -1,50 +1,57 @@
 #' species_list
 #' 
 #' Return the a species list given a taxonomic group
+#' @param SuperClass Request all species of this Superclass
 #' @param Class Request all species in this taxonomic Class
 #' @param Order Request all species in this taxonomic Order
 #' @param Family Request all species in this taxonomic Family
-#' @param SubFamily Request all species in this taxonomic SubFamily
+#' @param Subfamily Request all species in this taxonomic SubFamily
 #' @param Genus Request all species in this taxonomic Genus
 #' @param Species Request all species in this taxonomic Species
 #' @param SpecCode Request species name of species matching this SpecCode
-#' @param SpeciesRefNo Request species name of all species matching this SpeciesRefNo
-#' @param all_taxa The data.frame of all taxa used for the lookup. By default will be loaded
-#'  from cache if available, otherwise must be downloaded from the server; about 13 MB, may be 
-#'  slow.
-#' 
-#' @details The first time the function is called it will download and cache the complete
-#' @importFrom dplyr select_ %>%
-#' @importFrom lazyeval interp
-#' @importFrom tidyr unite_
+#' @param server fishbase or 
 #' @examples
-#' \dontrun{
+#' \donttest{
```

Comment:
what's the goal with changing dontrun to donttest? afaik `donttest` egs are run on cran checks, maybe that's what you're going for?

Reply:
yeah, I'm not 100% clear on it but that's basically what I'm going for.  I need some kind of `dont` there because the command can take > 5 s (at least the first time), but otherwise wanted it to run as part of the regular test.